cli/cmd/install/install_test.go

```diff
@@ -53,13 +53,14 @@ func TestCheckForPreviousSecrets(t *testing.T) {
 	c.kubernetes = fake.NewSimpleClientset()
 	secret := &v1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "test-consul-bootstrap-acl-token",
+			Name:   "test-consul-bootstrap-acl-token",
+			Labels: map[string]string{"managed-by": "consul-k8s"},
 		},
 	}
 	c.kubernetes.CoreV1().Secrets("default").Create(context.Background(), secret, metav1.CreateOptions{})
 	err := c.checkForPreviousSecrets()
 	require.Error(t, err)
-	require.Contains(t, err.Error(), "found consul-acl-bootstrap-token secret from previous installations: \"test-consul-bootstrap-acl-token\" in namespace \"default\". To delete, run kubectl delete secret test-consul-bootstrap-acl-token --namespace default")
+	require.Contains(t, err.Error(), "found consul secret from previous installations: \"test-consul-bootstrap-acl-token\" in namespace \"default\". To delete, run kubectl delete secret test-consul-bootstrap-acl-token --namespace default")
```

Comment:
You might be better off only including the relevant snippet of the error you want to check. Maybe "found Consul secret from previous installations". The longer the string that you are checking, the higher the likelihood that something in your implementation changes which breaks this test even when the "behavior" of the code, throwing an error when a secret is found, hasn't changed.

In Go, it is somewhat necessary to check the text of the error because there is only one error type (I personally think this was a silly thing to do). However, when you work with languages that have different error types, it's usually better to check just the type of the error and not the text of the error.